### ISSUE TITLE
trial man page incorrectly explains --coverage behaviour

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2309]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#2309|
|Type|defect|
|Created|2006-12-20 05:48:59Z|

Currently, it says:
```
 Generate  coverage  information in the given directory (relative
 to _trial_temp). Requires Python 2.3 or higher.
```

At the moment, trial doesn't let you set the coverage directory. It is always _trial_temp/coverage/. The man page should also have a stab at explaining what to do with the coverage output.

<details><summary>Searchable metadata</summary>

```
trac-id__2309 2309
type__defect defect
reporter__jml jml
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__ 
time__1166593739000000 1166593739000000
changetime__1166986280000000 1166986280000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

Trial should _also_ let you set the coverage directory, in addition to the immediate bugfix for the docs.  Is there a ticket for that?  It should be linked here.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_2"></a>|@jml commented|
|-|-|

No there isn't. Please file one.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

Um.  Don't file that ticket, please.  Trial _used_ to let you set the coverage directory, that's why the documentation says it does.  It doesn't let you do that anymore and I can't think of any reason for it to.  If you can, please write up an extensive justification so that 6 months from now someone doesn't decide it would be a good idea to delete it again.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun removed owner|
|-|-|

Doc fix is ready for review in coverage-docs-2309

**Comment by twisted-trac** [comment]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)<a name="note_5"></a>|@jerub commented|
|-|-|

Excellent. Please merge this documentation change.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [19065]) Merge coverage-docs-2309

Author: exarkun
Reviewer: jerub
Fixes [#2309](https://github.com/twisted/twisted/issues/2309)

Correct docstring and man page documentation of trial's --coverage
option so that it does not incorrectly suggest that a parameter is
accepted by the option which specifies the coverage output directory.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
